RPS_ver2.py

```diff
@@ -1,10 +1,16 @@
+import sys
+
 # program evaluates a two player's input for rock paper scissors and outputs the winners
 
-SET_PLAYS = set(['rock', 'paper', 'scissors'])
+player_one = "Player One"
```

Comment:
General rule of thumb; if you type the same string twice, it's time to put it in a variable